pkgs/applications/science/astronomy/siril/default.nix

```
@@ -1,27 +1,27 @@
-{ lib, stdenv, fetchFromGitLab, fetchFromGitHub, pkg-config, meson, ninja,
-  git, criterion, wrapGAppsHook, gtk3, libconfig, gnuplot, opencv,
-  fftwFloat, cfitsio, gsl, exiv2, curl, librtprocess, ffmpeg,
+{ lib, stdenv, fetchFromGitLab, pkg-config, meson, ninja, wrapGAppsHook,
+  git, criterion, gtk3, libconfig, gnuplot, opencv, json-glib,
```

```suggestion
, git, criterion, gtk3, libconfig, gnuplot, opencv, json-glib
```
Please move the , to the beginning of the line.

---

Thanks for the review. I've made the requested change, marking as resolved.